cli/src/test/java/com/devonfw/tools/ide/io/FileAccessImplTest.java

```diff
@@ -0,0 +1,227 @@
+package com.devonfw.tools.ide.io;
+
+import static org.junit.jupiter.api.Assertions.*;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.LinkOption;
+import java.nio.file.Path;
+import java.nio.file.Paths;
+import java.util.function.Function;
+
+import org.assertj.core.api.Assertions;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import com.devonfw.tools.ide.context.IdeContext;
+import com.devonfw.tools.ide.context.IdeTestContextMock;
+
+public class FileAccessImplTest extends Assertions {
+
+  @Test
+  void testSymlink(@TempDir Path tempDir) {
```

Comment:
not necessary but we normally declare  Test methods as public, even to the default is public, just a consistency thing across tests

Reply:
okey:)